src/Aspire.Hosting/Dashboard/DashboardViewModelService.cs

```diff
@@ -220,47 +220,47 @@ private async Task ProcessEndpointChange(WatchEventType watchEventType, Endpoint
 
         foreach (var ownerReference in endpoint.Metadata.OwnerReferences)
         {
-            // Find better way for this string switch
-            switch (ownerReference.Kind)
+            if (ownerReference.Kind == Dcp.Model.Dcp.ContainerKind)
```

Comment:
This looks like a big change, but it is not--it is just replacing hard-coded DCP object kind values with common definitions.